pkg/sql/randgen/mutator.go

```diff
@@ -816,7 +816,8 @@ func postgresCreateTableMutator(
 								if castExpr, ok := funcExpr.Exprs[0].(*tree.CastExpr); ok {
 									referencedType := colTypes[castExpr.Expr.(*tree.UnresolvedName).String()]
 									isContextDependentType := referencedType.Family() == types.TimestampFamily ||
-										referencedType.Family() == types.OidFamily
+										referencedType.Family() == types.OidFamily ||
```

Comment:
(TimestampFamily is never context dependent :o)

Reply:
we don't need timestamptz here since that statement wouldn't have been generated, as it already doesn't work with ccrdb

this block is just about making the statement work in PG as well. for whatever reason, in postgres, date/timestamp without timezone casting to string _is_ context depdendent. maybe because the format is dependent on locale? idk